private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -99,7 +129,7 @@ class Designer extends React.Component<DesignerProps, State> {
     designer.monacoModuleLoaded(monaco);
 
     if (this.props.templateJSON) {
-      designer.setCard(this.props.templateJSON);
+      designer.setCard({...this.props.templateJSON});
```

Comment:
why did we have to do this again?

Reply:
This is for when the designer is opened the second time. It has an issue with binding with designer.setCard(). This makes sure that we create a new object from the props and not use any weird binding from setCard()